crates/breez-sdk/lnurl/migrations/sqlite/20251107_sender_comments.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE sender_comments
+(
+    payment_hash   TEXT NOT NULL PRIMARY KEY,
+    user_pubkey    TEXT NOT NULL,
+    sender_comment TEXT NOT NULL,
+    invoice_expiry BIGINT NOT NULL,
```

Comment:
What is the purpose of invoice_expiry at this context?

Reply:
Removed that field